### DISCUSSION TITLE
1.02更新失败

### DISCUSSION THREAD

**Discussion by actuallyyao** [rime/home#1595] [Q&A] [answered]
mac鼠须管0.16.2更新到1.0.2失败：
<img width="472" alt="image" src="https://github.com/rime/home/assets/103360331/4b7da1a3-f542-4936-9c94-f5cdc990b618">

**Comment by LEOYoon-Tsaw** [comment, marked as answer]
0.18是個分水嶺，0.16得手動更新，0.18以上可以自動更新

**Reply by actuallyyao** [reply]
谢谢
